src/main/java/com/redhat/devtools/lsp4ij/LanguageServerWrapper.java

```diff
@@ -472,7 +492,13 @@ private void removeStopTimer(boolean stopping) {
     }
 
     private void updateStatus(ServerStatus serverStatus) {
+        // If this is an "interesting" status change, increment the language server definition's modification tracker before firing events
```

Comment:
If the status has changed to something other than `null` or `none` -- which is used for "dummy" wrappers that shouldn't result in a change -- increment the language server definition's modification count.